cmd/reduce.go

```diff
@@ -72,8 +72,8 @@ which allow reducing huge rpm repos to a smaller problem set for debugging, remo
 
 	reduceCmd.Flags().StringArrayVarP(&reduceopts.in, "input", "i", nil, "primary.xml of the repository")
 	reduceCmd.Flags().StringVarP(&reduceopts.out, "output", "o", "debug.xml", "where to write the repository file")
-	reduceCmd.Flags().StringVarP(&reduceopts.fedoraBaseSystem, "fedora-base-system", "f", "fedora-release-container", "fedora base system to choose from (e.g. fedora-release-server, fedora-release-container, ...)")
```

Comment:
can you keep the `fedora-base-system` flag and deprecate it with something like `reduceCmd.Flags().MarkDeprecated("fedora-base-system", "Please use 'basesystem' instead.")`?